packages/components/rate/src/Rate.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <div class="ix-rate-wrap">
+    <div
+      v-for="(item, index) in rateCount"
+      :key="index"
+      :ref="
+        el => {
+          if (el) starRefs[index] = el
+        }
+      "
+      class="ix-rate-item"
+      :title="getTooltip(index)"
+      :style="{ cursor: disabled ? 'auto' : 'pointer' }"
+      @mousemove="handleMouseEnter(item, $event)"
+      @mouseleave="handleMouseLeave(item)"
+      @click="handleClick(item, $event)"
+    >
+      <div class="ix-rate-full">
+        <ix-icon :name="rateIcon" class="ix-rate-iconfont-main" :class="getIconClass(item)" />
+      </div>
+
+      <div v-if="showDecimalIcon(item)" class="ix-rate-half">
+        <ix-icon :name="rateIcon" class="ix-rate-iconfont-main ix-rate-half-icon" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { isArray, PropTypes, toNumber } from '@idux/cdk/utils'
+import { computed, defineComponent, onBeforeUpdate, ref, watchEffect } from 'vue'
+import { IxIcon } from '@idux/components/icon'
+import { useGlobalConfig } from '@idux/components/core/config'
+
+const HALF = 2
+
+export default defineComponent({
+  name: 'IxRate',
+  components: { IxIcon },
+  props: {
+    value: PropTypes.oneOfType([Number, String]).def(0),
+    count: PropTypes.oneOfType([Number, String]),
```

Comment:
```suggestion
    count: withUndefined<PropTypes.oneOfType([Number, String])>,
```

oneOfType 的时候，需要加一个 withUndefined